src/Engine/ProtoCore/FFI/FFIHandler.cs

```diff
@@ -29,6 +30,7 @@ abstract public class DLLModule
     {
         public abstract List<FFIFunctionPointer> GetFunctionPointers(string className, string name);
         public abstract FFIFunctionPointer GetFunctionPointer(string className, string name, List<ProtoCore.Type> argTypes, ProtoCore.Type returnType);
+        public virtual void ScanModule(bool isReflectionContext) { }
```

Comment:
this should have a summary here as well - since you're adding it to the abstract class, it't kind of a like a soft interface.